.github/workflows/build-dotnet.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v1
       - name: Download workflow artifact
-        uses: dawidd6/action-download-artifact@v2.14.0
+        uses: dawidd6/action-download-artifact@v2.17.0
```

Comment:
#1.  it seems `dawidd6/action-download-artifact` has known download flakiness ([issue 93](https://github.com/dawidd6/action-download-artifact/issues/93) which causes "Not Found" errors and was fixed as of 2.14.1